packages/pyarrow/test_pyarrow.py

```diff
@@ -0,0 +1,24 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide
+def test_read_write_parquet():
```

Comment:
```suggestion
@run_in_pyodide(packages=["pyarrow", "numpy", "pandas"])
def test_read_write_parquet(selenium):
```

Reply:
Is that just for clarity? `run_in_pyodide` calls `loadPackagesFromImports` internally, so it shouldn't strictly need it?

Reply:
Ah, I think it is explicitly needed because the tests were failing, because `pytest-pyodide` will need to know which packages are to be loaded.